libraries/lib-wave-track/CachingPlayableSequence.cpp

```diff
@@ -15,7 +15,7 @@ void FillBufferFromTrackBlockSequence(
       size_t toWrite = std::min(
          segment.GetSampleCount().as_long_long(),
          static_cast<long long>(bufferSize - written));
-      written += segment.Copy(buffer, toWrite);
+      written += segment.Copy(buffer + written, toWrite);
```

Comment:
I understand the bug fix commit and expect you will squash it after complete review.

Reply:
Can do that.